common/crypto_utils/avalon_crypto_utils/worker_signing.py

```diff
@@ -79,7 +80,7 @@ def sign_message(self, message_hash_bytes):
         Parameters :
             message_hash_bytes: Message hash to sign in bytes
         Returns :
-            signed message in bytes.
+            signed message in base64 encoded string.
```

Comment:
Does this function return der encoded bytes. Why is return value mentioned as string ?

Reply:
This function returns signature in bytes. Updated the function comment to reflect the same.